include/aikido/planner/RankedMetaPlanner.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef AIKIDO_PLANNER_RANKEDMETAPLANNER_HPP_
+#define AIKIDO_PLANNER_RANKEDMETAPLANNER_HPP_
+
+#include "aikido/planner/CompositePlanner.hpp"
+
+namespace aikido {
+namespace planner {
+
+class RankedMetaPlanner : public CompositePlanner
+{
+public:
+  // Documentation inherited.
+  trajectory::TrajectoryPtr plan(
+      const Problem& problem, Result* result = nullptr) override;
```

Comment:
We can mark `final` specifier here when we're sure any derived class don't override the virtual function. I just leave it open because we haven't implemented this class yet.